recipe/meta.yaml

```diff
@@ -48,6 +48,8 @@ test:
     - PIL
     - PIL.Image
     - PIL.ImageCms  # [not win]
+    # Testing _imaging to verify that it doesn't fail like reported in https://anaconda.atlassian.net/browse/PKG-5184
+    - PIL._imaging
```

Comment:
You could add a test command `python -c "from PIL.Image import core"`

Reply:
The problem was in `openjpeg 2.4.0=*_1` see https://anaconda.slack.com/archives/C02D68R4D0D/p1720646319721279?thread_ts=1719980913.368079&cid=C02D68R4D0D

Reply:
I added `python -c "from PIL.Image import core"`